charts/librespeed/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: 1.0.6-12
+description: Librespeed is a HTML5 webpage to test upload and download speeds
+name: librespeed
+version: 1.0.0
+keywords:
+  - Librespeed
```

Comment:
This is really minor, but can we make this instance lowercase (e.g. `librespeed`)

Reply:
All are resolved! That's what I get for using find/replace